CHANGELOG/CHANGELOG-1.4.md

```diff
@@ -0,0 +1,134 @@
+# Changelog
+
+## [Unreleased](https://github.com/diem/diem/tree/HEAD)
+
+[Full Changelog](https://github.com/diem/diem/compare/diem-core-v1.3.0...HEAD)
+
+**Notable changes:**
+
+**[Consensus]**
+
+- Added decoupled execution. See related PRs ([\#8616](https://github.com/diem/diem/pull/8616), [\#8626](https://github.com/diem/diem/pull/8626), [\#8743](https://github.com/diem/diem/pull/8743), [\#8754](https://github.com/diem/diem/pull/8754), [\#8774](https://github.com/diem/diem/pull/8774), [\#8784](https://github.com/diem/diem/pull/8784), [\#8701](https://github.com/diem/diem/pull/8701), [\#8806](https://github.com/diem/diem/pull/8806), [\#8792](https://github.com/diem/diem/pull/8792), [\#8653](https://github.com/diem/diem/pull/8653))
+- \[decoupled-execution\] Add a new API for the TSafetyRules trait [\#8657](https://github.com/diem/diem/pull/8657)
+
+**[Diem Framework]**
+
+- Rename DiemID move resources and rust structures to VASPDomain [\#8695](https://github.com/diem/diem/pull/8695)
+- \[read/write set analysis\] Fix join child data [\#8751](https://github.com/diem/diem/pull/8751)
+- \[read/write set analysis\] adding some useful tests [\#8706](https://github.com/diem/diem/pull/8706)
+- \[read/write set\] handle addresses flowing into/out of structs[\#8658](https://github.com/diem/diem/pull/8658)
+- \[diem framework\] remove `verify=false` from scripts [\#8708](https://github.com/diem/diem/pull/8708)
+- \[diem-framework\] Add support for unit testing in the Diem Framework [\#8788](https://github.com/diem/diem/pull/8788)
+- \[diem-framework\] \[stdlib\] Migrate to phantom types [\#8739](https://github.com/diem/diem/pull/8739)
+- \[diem-framework\]\[move-stdlib\] Use Named Addresses [\#8394](https://github.com/diem/diem/pull/8394)
+
+**[Excecutor]**
+
+- \[execution\] remove mutex after interior mutability [\#8727](https://github.com/diem/diem/pull/8727)
+- \[executor\] support interior mutability to enable concurrent execute [\#8721](https://github.com/diem/diem/pull/8721)
+
+
+**[JSON RPC/Client]**
+
+- \[verifying-client\] Add get\_account\_transaction\(s\) [\#8530](https://github.com/diem/diem/pull/8530)
+- \[verifying-client\] Add transaction submit and wait [\#8666](https://github.com/diem/diem/pull/8666)
+- \[verifying-client\] Support get\_metadata via new get\_event\_by\_version\_with\_proof API [\#8705](https://github.com/diem/diem/pull/8705)
+- client: increase wait\_for\_transaction timeout [\#8770](https://github.com/diem/diem/pull/8770)
+- jsonrpc-types: add default unknown variants to various enums [\#8646](https://github.com/diem/diem/pull/8646)
+
+**[Mempool]**
+
+- \[mempool\] Prevent double decrements of upstream peers metric [\#8676](https://github.com/diem/diem/pull/8676)
+
+**[Move]**
+
+- Updated to version 1.4 of Move: see the Move release notes for details.
+- Diem Framework version 3. Details in planning. TBD.
```

Comment:
I'm not sure what details are planned to list here, but regardless it seems like this bullet belongs in the Diem Framework section.